dev-libs/dbus-glib/dbus-glib-0.112.ebuild

```diff
@@ -42,9 +51,16 @@ multilib_src_configure() {
 		$(use_enable static-libs static)
 	)
 
-	# Use host tool when building cross.
-	if ! use cros_host; then
-		myconf+=("--with-dbus-binding-tool=/usr/bin/dbus-binding-tool")
+	# Configure a CBUILD directory to make a native build tool.
+	if tc-is-cross-compiler; then
+		mkdir "${BUILD_DIR}-build" || die
+		cd "${BUILD_DIR}-build" || die
+		ECONF_SOURCE="${S}" econf_build
+		myconf+=( --with-dbus-binding-tool="$PWD/dbus/dbus-binding-tool" )
```

Comment:
So we do not need to keep the Flatcar [patch](https://github.com/flatcar-linux/coreos-overlay/commit/3a7d063c729e) for dbus-binding-tool, because Gentoo ebuild is already adding the `--with-dbus-binding-tool` option.
Nice.

Reply:
True. But then another issue popped up and I still needed to patch the ebuild. :) I'll add upstreaming the changes I have made to gentoo to my TODO list. Maybe at some point we will end up moving dbus-glib into portage-stable even.

Even better would be to port update_engine to use D-Bus functionality implemented in GLib instead, so dbus-glib could be removed, but this is a story for another time.